server/views/pages/confirmArrival.njk

```diff
@@ -59,7 +59,7 @@
             <div>
                 {{ govukButton({
                     classes: "govuk-button govuk-!-margin-bottom-6",
-                    href: "/prisoners/" + data.id + "/imprisonment-status",
+                    href: "/prisoners/" + data.id + "/sex",
```

Comment:
should the href be dependant on sex info in basm . I.e will we always go to /sex even if basm shows male or female?